src/wazuh_db/wdb.h

```diff
@@ -51,6 +51,15 @@
 #define AGENT_CS_ACTIVE          "active"
 #define AGENT_CS_DISCONNECTED    "disconnected"
 
+/// Enumeration of agents disconected status reasons.
+typedef enum agent_status_code_t {
+        INVALID_VERSION = 1,    ///< Invalid agent version
+        HC_SHUTDOWN_RECV,       ///< Shutdown message received
+        NO_KEEPALIVE,           ///< Disconnected because no keepalive received
+        RESET_BY_MANAGER,       ///< Connection reset by manager
+        ERR_VERSION_RECV        ///< Error retrieving version
```

Comment:
Just to keep this enum more organized, move this status code below `INVALID_VERSION` since these code are related to the same.